samples/sql-chatbot/pom.xml

```diff
@@ -15,9 +15,10 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-        <quarkus.platform.version>3.15.2</quarkus.platform.version>
+        <quarkus.platform.version>3.17.5</quarkus.platform.version>
```

Comment:
I prefer not use Quarlis LTS unless there is something the sample needs for the latest versions